Mlem/App/Views/Root/ContentView.swift

```diff
@@ -58,7 +58,9 @@ struct ContentView: View {
     var content: some View {
         CustomTabView(selectedIndex: $selectedTabIndex, tabs: [
             CustomTabItem(title: "Feeds", image: Icons.feeds, selectedImage: Icons.feedsFill) {
-                NavigationLayerView(layer: .init(root: .feeds, model: navigationModel), hasSheetModifiers: false)
+                NavigationSplitRootView(layer: .init(root: .feeds, model: navigationModel)) {
+                    SubscriptionListView()
+                }
```

Comment:
Do we maybe want to handle this using a `MultiplatformView` and only use the split view on iPad? It caused all sorts of nasty little fringe navigation behaviors in the v1 codebase, though hopefully the new tab reselection/navigation architecture will avoid those.

Reply:
Sure, makes sense 👍